formatTest/unit_tests/expected_output/bucklescript.re

```diff
@@ -90,7 +90,9 @@ let c = {
 let d = {
   "a": a2,
   "b": b,
-  "func": a => {"a": (arg1, arg2) => arg1 + arg2}
+  "func": a => {
+    "a": (arg1, arg2) => arg1 + arg2
+  }
```

Comment:
Seems missing here

Reply:
Yes JS objects were the only ones I didn’t get to. I’ll try to add them before landing.